File: js/find-email-domain.js
function findEmailDomain(address){
    let regExpDomain=/@([A-Za-z0-9\-]+\.)+[A-Za-z]{2,}/
    
    let domainFound=address.match(regExpDomain)
    
    return domainFound[0].slice(1)

}

let address = "prettyandsimple@example.com"

console.log(`The email domain in the addres ${address} is :`, findEmailDomain(address))

File: go/find-email-domain/find-email-domain.go
package main

import(
	"fmt"
	"regexp"
)

func findEmailDomain(address string) string{
	regExpDomain := regexp.MustCompile(`@([A-Za-z0-9\-]+\.)+[A-Za-z]{2,}`)
	domainFound := regExpDomain.FindString(address)
	return domainFound[1:]
}

func main(){
	address := "prettyandsimple@example.com"

	fmt.Printf("The email domain in the addres %s is : %s", address, findEmailDomain(address))
}